Automattic/wp-calypso
# Hosting Panel: Update UI for cards and add clipboard button

### Summary

Per [design feedback](https://github.com/Automattic/wp-calypso/pull/36442#issuecomment-542768438), this PR updates the UI for the cards in the Hosting panel, as well as adding a clipboard button in order to copy user login information.

<hr>

### Screenshots

**When no SFTP user is available**
<img width="1056" alt="Screen Shot 2019-10-17 at 12 12 59 PM" src="https://user-images.githubusercontent.com/8892849/67027849-4bfd3b80-f0d8-11e9-83be-77d97caeb287.png">

**When an SFTP user is available**
<img width="1062" alt="Screen Shot 2019-10-17 at 12 12 38 PM" src="https://user-images.githubusercontent.com/8892849/67027912-6d5e2780-f0d8-11e9-9595-d2c72c3562fa.png">

**When an SFTP user is created or the password is reset**
<img width="1053" alt="Screen Shot 2019-10-17 at 12 12 47 PM" src="https://user-images.githubusercontent.com/8892849/67027899-68997380-f0d8-11e9-8028-c6af81627e12.png">


<hr>

### Testing

* Spin up a local Calypso development environment.
* Apply D33899-code to your sandbox.
* Go to `http://calypso.localhost:3000/hosting/your.atomic.site`.
* Ensure that UI states reflected in the screenshots are visible.
* Ensure that 'Copy' buttons work as expected.
